chainConfigs/1.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
looks good! the deploy script still needs to be updated to read from this, right?

Reply:
yep!